server/events/vcs/github_client_test.go

```diff
@@ -1024,9 +1024,9 @@ func TestGithubClient_SplitComments(t *testing.T) {
 	err = client.CreateComment(repo, pull.Num, comment, "")
 	Ok(t, err)
 
-	body := strings.Split(githubComments[1].Body, "\n")
+	body := strings.Split(githubComments[0].Body, "\n")
 	firstSplit := strings.ToLower(body[0])
-	body = strings.Split(githubComments[3].Body, "\n")
+	body = strings.Split(githubComments[2].Body, "\n")
```

Comment:
This changes all vcs behavior. You mentioned bitbucket in the pr body. Could you add a test for bitbucket to check the comment order?